src/components/Resources/Resources.tsx

```diff
@@ -1,42 +1,35 @@
 import Image from "next/image";
 
-import { Button } from "../buttons/Button/Button";
+import { Button } from "@/components";
 
-const Resources = () => {
-  const articles = [
-    {
-      src: "/assets/images/eid7.jpg",
-      title: "key design lessons from lucy cunningham's...",
-      href: "/",
-      continue: "continue reading",
-    },
-    {
-      src: "/assets/images/eid8.jpg",
-      title: "english country-house style and the privelege of...",
-      href: "/",
-      continue: "continue reading",
-    },
-    {
-      src: "/assets/images/eid11.jpg",
-      title: "how to create a fabric scheme for your interior...",
-      href: "/",
-      continue: "continue reading",
-    },
-  ];
+import { articles } from "./data";
+
+interface ResourcesProps {
+  showHeader?: boolean;
+  articleFilter: string;
+}
+
+const Resources = ({ showHeader = true, articleFilter }: ResourcesProps) => {
+  const filteredArticles = articles.filter(
```

Comment:
Again, not sure if needed for now. We can always set a default value for articleFilter for time being and leave rest as is